ImageGoNord/GoNord.py

```diff
@@ -216,31 +216,30 @@ def open_image(self, path):
         """
         return Image.open(path)
 
-    def resize_image(self, image, w=0, h=0):
+    def resize_image(self, image, size=(0, 0)):
```

Comment:
Stavo per dirti che questa modifica avrebbe compromesso il servizio, ma hai già eseguito l'upgrade.

Reply:
potevi farmi il reject, non e' obbligatorio che se faccio qualcosa me la devi mergiare a forza

Reply:
mi sembrava inutile aprire un issue per discutere prima di una cosa che facevo prima a scrivere a codice e farvi il pull-request

Reply:
Son d'accordo con il fatto che non devo mergiare per forza, quando ho detto che hai eseguito già l'upgrade intendevo su imagegonord non pip quindi va bene :D

Reply:
ah ok perfetto
ho fixato il bug ma non ho mergiato la modifica del size (per adesso) anche se l'ho fatta sul mio repo, ti faccio il pull request?

Reply:
yes

Reply:
In teoria finché il servizio rimane col la 0.0.8, questo problema non si verifica.
Possiamo quindi programmare una release sull'API di Heroku, ma aspetterei a farla poiché prima vorrei apportare delle importanti migliorie sulle performance